tox.ini

```diff
@@ -1,22 +1,31 @@
 [tox]
-#envlist = py35-lint,py34-lint,py33-lint,py27-lint,py35-unit,py34-unit,py33-unit,py27-unit
-envlist = py27-lint, py27-unit, py35-mypy, py27-pipconflictchecker
+envlist =
+  py{27,33,34,35,36}-lint,
+  py{27,33,34,35,36}-unit,
+  py35-mypy,
+  py27-pipconflictchecker
+
 skipsdist = True
+skip_missing_interpreters = True
 
 [tox:travis]
 2.7 = py27
-3.5 = py35-mypy
+3.3 = py33
+3.4 = py34
+3.5 = py35
+3.6 = py36
 
 [testenv]
-deps = -rrequirements.txt
-
-[testenv:py35-py2_mypy]
-commands = make mypy
-whitelist_externals = make
 deps =
-  mypy==0.470
-  typed-ast==0.6.3
   -rrequirements.txt
```

Comment:
Earlier the tests were passing because ``six`` was being satisfied as a requirement of some other package. I will add it as an explicit dependency now. 
I believe I should add in ``setup.py`` rather than ``requirements.txt`` because it is required beyond testing.